packages/eui/src/components/form/described_form_group/described_form_group.stories.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { EuiForm } from '../form';
+import { EuiFormRow } from '../form_row';
+
+import {
+  EuiDescribedFormGroup,
+  EuiDescribedFormGroupProps,
+} from './described_form_group';
+import { EuiFieldText } from '../field_text';
```

Comment:
[nit] Minor preference more more relative to less relative
```suggestion
import { EuiFormRow } from '../form_row';
import { EuiFieldText } from '../field_text';

import {
  EuiDescribedFormGroup,
  EuiDescribedFormGroupProps,
} from './described_form_group';
```